OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/operations/impl/executors/LoginUserOperationExecutor.kt

```diff
@@ -99,6 +93,8 @@ internal class LoginUserOperationExecutor(
     private suspend fun createUser(createUserOperation: LoginUserOperation, operations: List<Operation>): ExecutionResponse {
         var identities = mapOf<String, String>()
         var subscriptions = mapOf<String, SubscriptionObject>()
+        val properties = mutableMapOf<String, String>()
+        properties["timezone_id"] = TimeUtils.getTimeZoneId()!!
```

Comment:
I would prefer to remove `!!` and change `TimeUtils.getTimeZoneId()` to return a non-optional String. That method seems like it should always return a String.

Reply:
Fixed! Ya that method always returns a string so I changed its return type to be non-optional